src/py/_pyodide/_core_docs.py

```diff
@@ -124,6 +125,81 @@ def to_bytes(self) -> bytes:
         an ArrayBuffer view.
         """
 
+    def to_file(self, file: IOBase):
+        """Writes the entire buffer to a file.
+
+        Will write the entire contents of the buffer to the current position of
+        the file.
+
+        Present only if the wrapped Javascript object is an ArrayBuffer or an
+        ArrayBuffer view.
+
+        Example
+        ------------
+        >>> import pytest; pytest.skip()
+        >>> from js import Uint8Array
+        >>> x = Uint8Array.new(range(10))
+        >>> with open('file.bin', 'wb') as fh:
+        ...    x.to_file(fh)
+        which is equivalent to,
+        >>> with open('file.bin', 'wb') as fh:
+        ...    data = x.to_bytes()
+        ...    fh.write(data)
+        but the latter copies the data twice whereas the former only copies the
+        data once.
+        """
+
+    def from_file(self, file: IOBase):
+        """Reads from a file into the buffer.
+
+        Will try to read a chunk of data of size the length of the buffer from
```

Comment:
It's slightly hard to parse with so many `of`

Maybe,
```suggestion
        Will try to read a chunk of data corresponding to the length of the buffer from
```